### PR TITLE
switch to pnpm

### PR DESCRIPTION
**Pull request by sachinraja**
part of #2708

## 🎯 Changes

What changes are made in this PR? Is it a feature or a bug fix?
Switch to pnpm from Yarn. Installations are much faster.

### CODE REVIEW

File: .github/workflows/examples-old-nodes.yml [hunk 1]
```diff
@@ -26,26 +26,32 @@ jobs:
...
+      - uses: actions/setup-node@v3
         with:
-          node-version: 16.x
+          node-version: 16
+          cache: 'pnpm'
+
       - run: node -v
-      - uses: bahmutov/npm-install@v1
+      - name: Install deps and build (with cache)
+        run: pnpm install
```
[the remark this answers is not in the document]
Comment (sachinraja) [reply .github/workflows/examples-old-nodes.yml:39]: `actions/setup-node` does this automatically with `cache: 'pnpm'`

**Review by juliusmarminge — APPROVED** [review]

File: .github/workflows/examples-old-nodes.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ jobs:
       matrix:
         dir: [.interop/next-prisma-starter]
         # start example with:
-        node-start: ['14.x']
+        node-start: [14]
```
Comment (KATT) [inline .github/workflows/examples-old-nodes.yml:17]:
```suggestion
        node-start: ['14.x']
```

File: .github/workflows/examples.yml [hunk 3]
```diff
@@ -38,7 +38,7 @@ jobs:
             # next-prisma-todomvc,
             standalone-server,
           ]
-        node-start: ['16.x']
+        node-start: [16]
```
Comment (KATT) [inline .github/workflows/examples.yml:41]:
```suggestion
        node-start: ['16.x']
```

File: .github/workflows/lint.yml [hunk 4]
```diff
@@ -6,20 +6,31 @@ concurrency:
...
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
 
-      - uses: actions/setup-node@v1
+      - uses: pnpm/action-setup@v2.2.2
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
```
Comment (KATT) [inline .github/workflows/lint.yml:21]:
```suggestion
        node-start: ['16.x']
```

File: .github/workflows/examples-old-nodes.yml [hunk 5]
```diff
@@ -26,26 +26,38 @@ jobs:
...
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+
+      - uses: pnpm/action-setup@v2.2.2
+
+      - uses: actions/setup-node@v3
         with:
-          node-version: 16.x
+          node-version: 16
```
Comment (KATT) [inline .github/workflows/examples-old-nodes.yml:34]:
```suggestion
          node-version: 16.x
```

File: .github/workflows/examples.yml [hunk 6]
```diff
@@ -50,30 +50,50 @@ jobs:
           - 5432:5432
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+
+      - uses: pnpm/action-setup@v2.2.2
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
```
Comment (KATT) [inline .github/workflows/examples.yml:58]:
```suggestion
          node-version: 16.x
```

**Review by KATT — APPROVED** [review]

**Pull request merged**